icicle-compiler/icicle.cabal

```diff
@@ -88,7 +88,7 @@ library
                      -- a bit cleaner.
                      -- As an added benefit, parsec will give us nicer error messages.
                      , attoparsec                      >= 0.12       && < 0.14
-                     , megaparsec                      == 5.0.*
+                     , megaparsec                      == 5.1.*
```

Comment:
Can we just increase the upper bound here? bumping the lower bound could break things that depend on icicle.

```megaparsec                      >= 5.0        && < 5.2```

Reply:
I'll do that